src/common/table.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-disable @typescript-eslint/no-explicit-any, class-methods-use-this */
+
+import { CliUx } from '@oclif/core';
+
+export function displayTable(
+  rows: any[],
+  title: string,
+  columns: CliUx.Table.table.Columns<any>,
+  options?: CliUx.Table.table.Options
+): void {
+  CliUx.ux.log();
```

Comment:
What does this line do?

Reply:
Adds an empty line, just to separate the final result from command line